src/stories/Motion.mdx

```diff
@@ -0,0 +1,78 @@
+import { Meta, Story, Canvas, Controls, Stories } from '@storybook/blocks';
+import * as MotionStories from './Motion.stories.js';
+
+<Meta of={MotionStories} />
+
+# Motion
+
+[Figma Design Source](https://www.figma.com/file/5TqtPa7KWfhJbQv6ELnbqf/Foundation?node-id=72%3A17784&mode=dev)
+
+## Example
+
+<Canvas>
+  <Story of={MotionStories.Transition} />
+</Canvas>
+
+### Controls
+
+<Controls of={MotionStories.Transition} />
+
+## Usage
+
+### SCSS Mixins
+
+```css
+@use '@kyndryl-design-system/shidoka-foundation/scss/mixins/motion.scss';
+$transitionPropsVar: background-color, border;
+
+.your-css-selector {
+  @include motion.transition(
+    $transitionPropsVar,
+    var(--kd-anim-duration-med),
+    var(--kd-anim-easing-in)
+  );
+}
+
+.your-css-selector {
+  @include motion.transition($transitionPropsVar, var(--kd-anim-duration-med));
+}
+
+.your-css-selector {
+  @include motion.transition($transitionPropsVar);
+}
+
+.your-css-selector {
+  @include motion.transition(
+    $transitionPropsVar,
+    150ms,
+    cubic-bezier(0.1, 0.7, 1, 0.1)
+  );
+}
+```
+
+### CSS Variables
+
+```css
+@use '@kyndryl-design-system/shidoka-foundation/dist/scss/variables/motion.scss';
```

Comment:
This doesn't actually need to be included for the CSS variables. All CSS vars are included with the root css file.

Reply:
Removed